components/help-dropdown/help-dropdown.jsx

```diff
@@ -23,7 +23,7 @@ export default class HelpDropdown extends React.Component {
     });
 
     return (
-      <select className="custom-select w-100" value={ this.props.value } onChange={ (event) => {this.props.helpType(event);} }>
+      <select className={`custom-select w-100 ${this.props.specialClass}`} value={ this.props.value } onChange={ (event) => {this.props.helpType(event);} }>
```

Comment:
class `help-dropdown` should always be here. We shouldn't need `specialClass` prop to determine it.